reference/twig-extensions/functions/_page_structure.inc

```diff
@@ -8,7 +8,6 @@
     - **changer**: User ID of changer
     - **created**: Date of creation
     - **creator**: User ID of creator
-    - **localizations**: Localizations including URLs to other languages
```

Comment:
`localizations` was never part of the `sulu_content_load` response. it is added in the [`ParameterResolver`](https://github.com/sulu/sulu/blob/2.5/src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php) which is only used by the `WebsiteController`, but not the `sulu_content_load` extension.